client/components/UserAccount.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import store, { updateAccountThunk, deleteUserThunk } from '../store';
+
+class UserAccount extends React.Component {
+  constructor(props) {
+    super(props)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  handleSubmit(event) {
+    event.preventDefault()
+    const user = {
+      firstName: event.target.firstName.value,
+      lastName: event.target.lastName.value,
+      email: event.target.email.value,
+      address: event.target.address.value
+    }
+    this.props.handleUpdateUser(user)
+  }
+
+
+  render() {
+    const { user } = this.props
+
+    return (
+    <div>
+      <form onSubmit = {this.handleSubmit}>
```

Comment:
functionally the form is ok, but we should make this look better if we have time.